src/frontend/komponenter/Fagsak/Vedtak/RefusjonEøs/RefusjonEøsPeriode.tsx

```diff
@@ -72,17 +68,26 @@ const RefusjonEøsPeriode: React.FC<IRefusjonEøsPeriode> = ({ refusjonEøs, beh
             onOpenChange={håndterLukkingOgÅpningAvPanel}
             content={
                 <FlexColumnDiv>
-                    <RefusjonEøsSkjema skjema={skjema} />
+                    <RefusjonEøsSkjema
+                        skjema={skjema}
+                        key={erRadEkspandert ? 'ekspandert' : 'lukket'}
```

Comment:
key brukes for å trigge rerender av skjemaet når det åpnes/lukkes

Reply:
Nå er ikke denne unik på tvers av radene. Kan det bli et problem?

Reply:
✅